### ISSUE TITLE
Version ZIP archive installation for OSX

### ISSUE THREAD

**Issue by jvanzyl** [aws/aws-cli#5134]
As a user of OSX and Linux it would be nice to be able to install the OSX and Linux versions the same way. Installing a developer tool on OSX would be most preferable, I think, to developers being a ZIP archive or available via brew. But for the sake of consistency, it would be great if a ZIP was available.

**Comment by KaibaLopez** [comment]
Hi @jvanzyl , I think this is a duplicate issue from #5123 
but I can see that you already posted there so... is there something I'm missing here? How is this feature request different?

**Comment by jvanzyl** [comment]
I was asked to create a new issue. All I'm looking for is a versioned ZIP for OSX as the PKG is not very amenable to automation.

**Comment by KaibaLopez** [comment]
ahh I see... 
We are kind of blocked on this, We don’t do a zip because of the uncertain future of distributing binaries directly.
Not to say that this won't be done in the future.

**Comment by jvanzyl** [comment]
Not sure what that means, but it's easy enough for me to make my own binary and publish it.

**Comment by skyzyx** [comment]
Firstly, this would be solved by https://github.com/aws/aws-cli/issues/5578.

Secondly, two packaging formats can be notarized by Apple — `.zip` and `.dmg`. You're doing the latter, but not the former. See [mitchellh/gon](https://github.com/mitchellh/gon) for tooling and more information.

**Comment by skyzyx** [comment]
Cross-linking https://github.com/aws/aws-cli/issues/4926

**Comment by tim-finnigan** [comment]
This looks like a duplicate of https://github.com/aws/aws-cli/issues/4926. I think we should close this and track the request there.
